### DISCUSSION TITLE
Ekkko Station for Plug Data is available for download

### DISCUSSION THREAD

**Discussion by pereshaped** [plugdata-team/plugdata#1874] [General]
Hi, you can download from my blog here:
https://tinyurl.com/EkkkoMn

**Comment by dromer** [comment]
Small reminder that the CC BY-NC-SA license does not conform with Free Software and in general is a poor fit for software projects.

**Reply by pereshaped** [reply]
Why and what do you recommend?

**Reply by dromer** [reply]
The NC clause does not conform with Free Software guidelines.

It will depend on your intention what license to use.
If you intend to have an NC clause then it is not Free Software under the OSI definition.

**Reply by dromer** [reply]
Creative Commons themselves recommend against using their licenses for software: https://creativecommons.org/faq/#can-i-apply-a-creative-commons-license-to-software

**Comment by pereshaped** [comment]
You still do not offer an alternative. I'll keep the license and remove the open source definition until Im clearer wih this. I'll remove the open source tag.

**Reply by dromer** [reply]
I can't pick a license for you if you don't give any specific requirements.
There is a huge list of FOSS licenses to choose from: https://opensource.org/licenses

**Reply by pereshaped** [reply]
That is what I needed to look at ;-)
